pandas/tests/apply/test_frame_apply.py

```diff
@@ -1182,7 +1182,7 @@ def test_agg_multiple_mixed_raises():
 
     # sorted index
     # TODO: GH#49399 will fix error message
```

Comment:
is the TODO in this comment addressed? if so, can you remove it

Reply:
Yes, this is addressed and removed it.